src/suricata.c

```diff
@@ -2687,6 +2690,12 @@ int PostConfLoadedSetup(SCInstance *suri)
 
     SetMasterExceptionPolicy();
 
+    int b;
+    int ret = ConfGetBool("stats.eps-per-app-proto-errors", &b);
+    if (ret) {
+        g_stats_eps_per_app_proto_errors = (b == 1);
+    }
+
```

Comment:
Probably not the best solution, but must get this config before app-layer counters setup.